.github/workflows/FT-das.yml

```diff
@@ -0,0 +1,240 @@
+name: Functional Tests
+
+on:
+  workflow_dispatch:
+    inputs:
+      api_version:
+        description: "API Version"
+        default: "latest"
+        type: string
+        required: False
+      worker_version:
+        description: "Worker Version"
+        default: "latest"
+        type: string
+        required: False
+      cli_version:
+        description: "CLI Version"
+        default: "latest"
+        type: string
+        required: False
+  workflow_call:
+    inputs:
+      api_version:
+        description: "API Version"
+        default: "latest"
+        type: string
+        required: False
+      worker_version:
+        description: "Worker Version"
+        default: "latest"
+        type: string
+        required: False
+      cli_version:
+        description: "CLI Version"
+        default: "latest"
+        type: string
+        required: False
+    secrets:
+      RSTUF_ONLINE_KEY:
+        required: True
+env:
+  MAKE_FT_TARGET: functional-tests
+  REQUIREMENTS_PATH: requirements.txt
+
+jobs:
+  functional-das:
+    name: "FT|DAS|LocalStorage, LocalKeyVault"
+    runs-on: ubuntu-latest
+    services:
+      rabbitmq:
+        image: rabbitmq:3-alpine
+
+      redis:
+        image: redis:4.0
+
+      sql:
+        image: postgres:15.1
+        env:
+          POSTGRES_PASSWORD: secret
+
+      web-server:
+        image: httpd:latest
+        volumes:
+          - rstuf-storage:/usr/local/apache2/htdocs
+        ports:
+          - 8080:80
+
+      rstuf-worker-1:
+        image: ghcr.io/repository-service-tuf/repository-service-tuf-worker:${{ inputs.worker_version }}
+        volumes:
+          - rstuf-worker-data:/data
+          - rstuf-storage:/var/opt/repository-service-tuf/storage
+          - rstuf-keystorage:/var/opt/repository-service-tuf/keystorage
+        env:
+          RSTUF_STORAGE_BACKEND: LocalStorage
+          RSTUF_KEYVAULT_BACKEND: LocalKeyVault
```

Comment:
Do we plan to have similar workflows when testing the other services as well?

Reply:
Yes, but it will have other `FT-das-awss3.yml`, for example.